core/src/main/scala/org/apache/spark/h2o/backends/external/RestCommunication.scala

```diff
@@ -134,9 +134,9 @@ trait RestCommunication extends Logging {
         connection.getResponseCode()
       }
       statusCode match {
-        case HttpURLConnection.HTTP_OK => logInfo(
+        case HttpURLConnection.HTTP_OK => logDebug(
```

Comment:
Maybe I would leave there `logInfo` for a while and change it `logDebug` when logs become polluted with these messages.